lms/templates/instructor/instructor_dashboard_2/cohort-editor.underscore

```diff
@@ -1,63 +1,72 @@
-        </div>
+            <h4 class="form-title"><%- gettext('Add students to this cohort group') %></h4>
+
+            <div class="form-introduction">
+                <p><%- gettext('Note: Students can be in only one cohort group. Adding students to this group overrides any previous group assignment.') %></p>
+            </div>
 
-        <div class="cohort-confirmations"></div>
-        <div class="cohort-errors"></div>
+            <div class="cohort-confirmations"></div>
+            <div class="cohort-errors"></div>
 
-        <div class="form-fields">
-            <div class="field field-textarea is-required">
-                <label for="cohort-management-group-add-students" class="label">
-                    <%- gettext('Enter email addresses and/or usernames separated by new lines or commas for students to add. *') %>
-                    <span class="sr"><%- gettext('(Required Field)') %></span>
-                </label>
-                <textarea name="cohort-management-group-add-students" id="cohort-management-group-add-students"
-                          class="input cohort-management-group-add-students"
-                          placeholder="<%- gettext('e.g. johndoe@example.com, JaneDoe, joeydoe@example.com') %>"></textarea>
+            <div class="form-fields">
+                <div class="field field-textarea is-required">
+                    <label for="cohort-management-group-add-students" class="label">
+                        <%- gettext('Enter email addresses and/or usernames, separated by new lines or commas, for the students you want to add. *') %>
+                        <span class="sr"><%- gettext('(Required Field)') %></span>
+                    </label>
+                    <textarea name="cohort-management-group-add-students" id="cohort-management-group-add-students"
+                              class="input cohort-management-group-add-students"
+                              placeholder="<%- gettext('e.g. johndoe@example.com, JaneDoe, joeydoe@example.com') %>"></textarea>
 
-                <span class="tip"><%- gettext('You will not get notification for emails that bounce, so please double-check spelling.') %></span>
+                    <span class="tip"><%- gettext('You will not receive notification for emails that bounce, so double-check your spelling.') %></span>
+                </div>
             </div>
-        </div>
 
-        <div class="form-actions">
-            <button class="form-submit button action-primary action-view">
-                <i class="button-icon icon icon-plus"></i> <%- gettext('Add Students') %>
-            </button>
-        </div>
-    </form>
-</div>
+            <div class="form-actions">
+                <button class="form-submit button action-primary action-view">
+                    <i class="button-icon icon icon-plus"></i> <%- gettext('Add Students') %>
```

Comment:
When accessible text of equal value is adjacent to the icon (99% of cases here at edX) we should add ```aria-hidden="true"``` to the icon fonts.  Some screen readers (Like Voiceover) will focus on them and remain silent, which makes screen reader users wonder if they are missing important content.  It also visually renders as a generic glyph in screen readers that are also used by people with cognitive disabilities (like Voiceover).  This should do the trick:
```<i class="button-icon icon icon-plus" aria-hidden="true"></i>```  <--this is one of the very few exceptions for using aria-hidden at all
@clrux add this note to your 'bag-o-patterns' when you get around to it